dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -60,6 +60,13 @@ ams:
   #    username: root
   #    password: root
 
+  database:
+    type: mysql
+    jdbc-driver-class: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://10.122.173.167:3306/arctic_opensource_v5_ocean_paimon?useUnicode=true&characterEncoding=UTF8&autoReconnect=true&useAffectedRows=true&useSSL=false&zeroDateTimeBehavior=convertToNull
+    username: readwrite
+    password: 3221631
+
```

Comment:
Do not leak the password to Github.....

Reply:
If you need run test in intellij, please using a env variable `AMORO_CONF_DIR`

Reply:
ok